packages/contracts/contracts/foundry/SphinxConstants.sol

```diff
@@ -6,11 +6,11 @@ import { NetworkInfo, NetworkType } from "./SphinxPluginTypes.sol";
 contract SphinxConstants {
   address public constant compatibilityFallbackHandlerAddress = 0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4;
   address public constant multiSendAddress = 0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761;
-  address public constant sphinxModuleProxyFactoryAddress = 0x8f3301c9Eada5642B5bB12FD047D3EBb2932E619;
-  address public constant managedServiceAddress = 0xB5E96127D417b1B3ef8438496a38A143167209c7;
+  address public constant sphinxModuleProxyFactoryAddress = 0xb3C4df24Ac0e21F9436F9A6A6BB4F7566872D003;
+  address public constant managedServiceAddress = 0xE1FC160c3037553ec654822653681381a68AB4EF;
   address public constant safeFactoryAddress = 0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2;
   address public constant safeSingletonAddress = 0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552;
-  address public constant sphinxModuleImplAddress = 0x8f4E4d51B8050B0ff713eff1F88f3dD8b5e8a530;
+  address public constant sphinxModuleImplAddress = 0x3b148245BB8b75cF0FA501688bcE9Cbd0076Af4f;
```

Comment:
Curious to hear your thoughts on this. These addresses should not change b/c I did not change any of the core contract code. We also can't merge this PR with different addresses for these contracts b/c they will not correspond to the real deployed contracts. 

I was not able to figure out exactly why these changed. Have any idea?